app/assets/javascripts/miq_application.js

```diff
@@ -470,6 +470,14 @@ function miqSetButtons(count, button_div) {
       var button = $(v);
       miqButtonOnWhen(button.parent(), button.data('onwhen'), count);
     });
+  } else if (button_div.match("_buttons$")) { // Handle buttons that are part of miq toolbars
```

Comment:
@h-kataria : can you, please, fix the comment here?
seems it should say "... NOT part of toolbars..." 
plus add a reference to the RHN functionality.

With the comment unchanged someone will delete the code again :-(